hazelcast/src/main/java/com/hazelcast/client/impl/protocol/codec/holder/CacheConfigHolder.java

```diff
@@ -221,6 +230,13 @@ public boolean isDisablePerEntryInvalidationEvents() {
         config.setMergePolicyConfig(mergePolicyConfig);
         config.setDisablePerEntryInvalidationEvents(disablePerEntryInvalidationEvents);
 
+        if (cachePartitionLostListenerConfigs != null) {
+            List<CachePartitionLostListenerConfig> partitionLostListenerConfigs = new ArrayList<>(
+                    cachePartitionLostListenerConfigs.size());
+            cachePartitionLostListenerConfigs.forEach(listenerConfigHolder -> partitionLostListenerConfigs
+                    .add(listenerConfigHolder.asListenerConfig(serializationService)));
```

Comment:
It seems we create and fill a temporary `ArrayList` here that we drop after filling without using it. I guess we miss a `config.setPartitionLostListenerConfigs()` call here.

Reply:
Good catch. Fixed it.